### PR TITLE
initial setup and routes

### PR DESCRIPTION
**Pull request by aatishpandey**
initial setup
basic routing

### CODE REVIEW

**Review by coldpigli — APPROVED** [review]
Everything looks good.
Only a small suggestion to extract flex css attributes to a common class. Because you are repeating the flex code in a lot of places. Everything else looks great!

**Review by vaibhav-mahalle — APPROVED** [review]
just a suggestion upload packages and all unrelated code in a separate commit, other than that everything looks good, not much to review.

File: public/index.html [hunk 1]
```diff
@@ -24,12 +20,13 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
```
Comment (rahulcs754) [inline public/index.html:21]: remove comment code

**Review by rahulcs754 — COMMENTED** [review]
I checked you code and in this code you can remove your comment code.
 and try to export all component using index.js

**Pull request merged**